src/main/scala/com/codacy/client/stash/util/AvatarUtil.scala

```diff
@@ -0,0 +1,18 @@
+package com.codacy.client.stash.util
+
+object AvatarUtil {
+
+  /**
+    * Size in pixels to return avatars
+    */
+  val defaultAvatarSize = 64
+
+  /**
+    * Bitbucket Server accepts a parameter on most endpoints to include avatars in the response. Calling this method
+    * with true will add the parameter required, if false an empty map is returned.
+    */
+  def addAvatarToParams(includeAvatar: Boolean): Map[String, String] = {
```

Comment:
This signature seems a bit odd. I would either:
* Keep the name, and add the "params" to the method arguments and return the parameters with it
* Change the name (like "avatarParameter"), don't receive the boolean, and always generate the avatar parameter

What do you think?

Reply:
Yeah I thought about both of these options too. I didn't have a feeling on any of the three so I just kept it but if you're not able to sleep with what it is I would probably go with your first option.